### DISCUSSION TITLE
I get this error: "No renderer found ..."

### DISCUSSION THREAD

**Discussion by himmetozcan** [microsoft/vscode-jupyter#12415] [Questions and Answers]
When I try to plot something with matplotlib I get this error:
`No renderer found for "image/png"`

Also get this error for JSON files: 
`No renderer found for "application/vnd.jupyter.widget-view+json" `

Does anybody know what is going on?

**Comment by petercoles** [comment]
No idea why, but for me running "Developer: Reload Window" resolved the issue.

**Reply by eduardolfULPGC2003** [reply]
Really worked. Thank you!

**Reply by Anivader** [reply]
I will also add that make sure you're in the right virtual environment. Sometimes you get switched - I experienced this in VSCode.

**Comment by DonJayamanne** [comment]
@himmetozcan Sorry we missed this.
Does this issue still exist, if it does, please could you create an issue and we'll get you unblocked.
